### ISSUE TITLE
error

### ISSUE THREAD

**Issue by jackfrost168** [bravotty/SKNet-tensorflow#3]
gap = tflearn.global_avg_pool(net)
The input should be  fea_U，right？

**Comment by jackfrost168** [comment]
I have other two questions.
why do you use add() in att_vec = tf.add(att_vec, fcs)？
And you fuse different branchs before softmax?

**Comment by bravotty** [comment]
Thanks, now 'gap = tflearn.global_avg_pool(fea_U)'
The operation which i did is follow the origin paper, pls refer to original paper~ thx!
